loaddictionaries.cc

```diff
@@ -266,18 +266,17 @@ void loadDictionaries( QWidget * parent, bool showInitially,
 
   QStringList allIdxFiles = indexDir.entryList( QDir::Files | QDir::Dirs | QDir::NoDotAndDotDot | QDir::NoSymLinks );
 
-  for( QStringList::const_iterator i = allIdxFiles.constBegin(); i != allIdxFiles.constEnd(); ++i )
+  for( const auto & i: allIdxFiles)
   {
```

Comment:
I do not like the name variable `i` ,can you change it into some meaningful name ?

Reply:
fixed